xml/depl_maintenance.xml

```diff
@@ -508,6 +508,258 @@
    </itemizedlist>
   </sect2>
 
+  <sect2 xml:id="sec.depl.maintenance.postgre-mariadb.upgrade">
+   <title>Preparing &postgres;-Based &productname; 7 for Upgrade</title>
+   <para>
+    Upgrading &productname; 7 is only possible when it uses &mariadb; deployed
+    with the Database &barcl;. This means that before you can proceed with
+    upgrading &productname; 7, you must migrate &postgres; to &mariadb; first. The following description covers several possible scenarios.
+   </para>
+   <sect3 xml:id="sec.postgre-mariadb.upgrade.scenario1">
+    <title>Non-HA setup or HA Setup with a 2-Node Cluster and
```

Comment:
Hm, why is this mentioning 2-Node postgresql clusters explicitly? The instructions below are perfectly fine for 3-node postgresql clusters as well, no? (And there is no section covering >2 node clusters AFAICS)

Reply:
Rigth, this section should be called "Setup with more than 2 nodes". I think we've already cleared that, but seems like something got lost